src/components/YandexForm/YandexForm.tsx

```diff
@@ -112,7 +112,7 @@ const YandexForm = (props: YandexFormProps) => {
                     onLoad?.();
                 }
 
-                if (message === 'sent') {
```

Comment:
please add comment why we need to check redirectUrl

Reply:
done